mix.exs

```diff
@@ -48,4 +50,18 @@ defmodule StbImage.MixProject do
       links: %{"GitHub" => @github_url}
     ]
   end
+
+  defp make_executable() do
+    case :os.type() do
+      {:win32, _} -> "nmake"
+      _ -> "make"
+    end
+  end
+
+  defp make_makefile() do
+    case :os.type() do
+      {:win32, _} -> "Makefile.win"
+      _ -> "Makefile"
+    end
+  end
```

Comment:
I think those are the Elixir make defaults. If you remove them, it should still work. :)

Reply:
This (and the other line you mentioned below) actually caused some problems for windows users who are using mingw32-make, https://github.com/elixir-nx/stb_image/runs/7570792192?check_suite_focus=true.

I'll try to get mingw-* things installed on my windows and see how to make it compatible.

Reply:
I think we can remove those functions then if we are not supporting mingw?

Reply:
I left this because for systems that only have mingw32-make, elixir_make will try to use that, and the error message might be confusing? as it would be some error messages from the mingw32-make saying there are errors in the Makefile instead of nmake not found?

And when I was testing mingw32-make in msys2, it would require another Makefile (current `Makefile` is not compatible with msys2). So if we are going to support mingw32, we'll have something like

```elixir
  defp make_makefile() do
    case :os.type() do
      {:win32, _} -> 
        cond do
          System.find_executable("nmake") -> "Makefile.win"
          System.find_executable("make") -> "Makefile.mingw"
          true -> "Makefile.win"
        end
      _ -> "Makefile"
    end
  end
``` 

The full patch for mingw-* in msys2 is [here](https://github.com/elixir-nx/stb_image/pull/24/commits/3d913bceaca3d5f7822a94f86ce332473f36fc39). But that patch won't work with cmd or Powershell. ;;;;;